### DISCUSSION TITLE
Поле файл стопорит выгрузку в csv

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#175] [ОШБК!] [answered]
Если в таблице есть поле с файлом в табличной части, то показывает ошибку
"Экспорт CSV недоступен для поля file."

Приходится принудительно выключать отображение поля через глаз и выгружать

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Угу, надо автоматически исключить

**Reply by ttm-alexeypolunin** [reply]
**Проверьте в 3.6.45.5 и выше**

**Reply by erpic-ru** [reply]
ПРи импорте CSV тоже ограничение стреляет
